app/views/profiles/edit.mobile.haml

```diff
@@ -2,13 +2,14 @@
 -# licensed under the Affero General Public License version 3 or later. See
 -# the COPYRIGHT file.
 
-= render "shared/settings_nav"
+.stream_element
```

Comment:
Yeah I agree. I think that we should have two layouts in the mobile version, the pages which contain elements, like the stream, the profile, the notifications or the contacts pages, and the pages are more about text, like the settings pages, the admin panel, the getting started...

The pages of the first group should have the grey background and then a white bg around each elements, but the text pages should directly have a white background, to avoid losing width space and look overloaded.

What do you think about that? It's a bigger work than what I'm doing here, that's why I simply used `.stream_element` as a temporary solution, but I agree, temp can become long, so I can make this one cleaner.